tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/access/FieldAccessTest.java

```diff
@@ -52,8 +52,7 @@ public void testNegativeCases() {
         int i = 0;
         validateError(negativeResult, i++, "invalid operation: type 'Employee' does not support field access " +
                               "for non-required field 'id'", 32, 9);
-        validateError(negativeResult, i++, "invalid operation: type 'Employee' does not support field access " +
-                              "for non-required field 'salary'", 33, 9);
+        validateError(negativeResult, i++, "undefined field 'salary' in record 'Employee'", 33, 9);
```

Comment:
Shall we add negative tests for unions if we don't have them already.

Reply:
Added negative test cases for union of records as well.